main.py

```diff
@@ -25,10 +28,14 @@ def createTodaysGames(games, df):
     for game in games:
         home_team = game[0]
         away_team = game[1]
+        home_abrev = game[2]
+        away_abrev = game[3]
         todays_games_uo.append(input(home_team + ' vs ' + away_team + ': '))
```

Comment:
Remove and grab under / over (totals) as well.

Reply:
i think over/under need to be optional for user because each user will have a specific over/under total

Reply:
I think this would be the same case for team lines as well. Odds will be slightly different between books.